nl

```diff
@@ -0,0 +1,284 @@
+                "disclaimer_clickwrap_input_label": "Het beleid accepteren",
+                "disclaimer_clickwrap_error_required": "U moet de beleidsregels lezen en accepteren om verder te kunnen gaan",
+                "disclaimer_dialog_heading": "Accepteren om door te gaan",
+                "disclaimer_dialog_description": "Om door te gaan met inschrijven, moet u onze beleidsregels lezen en accepteren",
+                "disclaimer": "Door verder te gaan, gaat u akkoord met ons beleid",
+                "disclaimer_terms_of_use_link": "Gebruiksvoorwaarden",
+                "disclaimer_privacy_link": "Privacybeleid"
+            },
+            "sms_verify_code_page": {
+                "page_title": "Meervoudige authenticatie | ${business_name}",
+                "heading": "Telefoonnummer verifiëren",
+                "description": "Voer de code in die we zojuist naar ${phone_number] hebben gestuurd",
+                "code_input_label": "Code",
+                "code_input_error_expired_code": "Uw bevestigingscode is verlopen, vraag een nieuwe aan",
+                "code_input_error_incorrect_code": "Voer een geldige bevestigingscode in",
+                "code_input_error_incorrect_code_length": "De bevestigingscode moet uit 6 cijfers bestaan",
+                "code_input_error_resend_requested_too_soon": "Kan nog geen code versturen. Probeer het later opnieuw",
+                "confirmation_code_input_error_unknown_error": "Fout met de authenticatie",
+                "continue_button": "Verifieer",
+                "didnt_receive_a_code_description": "Geen code ontvangen?",
+                "didnt_receive_a_code_link": "De code opnieuw verzenden",
+                "choose_another_method_link": "Kies een andere methode"
+            },
+            "subscribe_form_widget": {
+                "page_title": "Inschrijven",
+                "heading": "Blijf op de hoogte",
+                "description": "Blijf op de hoogte van productupdates, nieuws en gebeurtenissen",
+                "name_input_label": "Voornaam",
+                "name_input_error_required": "Voer een naam in,",
+                "name_input_error_exceeds_max_length": "Naam is te lang,",
+                "email_input_label": "E-mailadres",
+                "email_input_error_invalid": "Voer een geldig e-mailadres in,",
+                "email_input_error_required": "E-mailadres is verplicht,",
+                "email_input_error_duplicate_email_found": "Dit e-mailadres is al ingeschreven,",
+                "continue_button": "Inschrijven",
+                "success_heading": "Succes! Welkom bij de berichtgeving",
+                "credit_powered_by": "Mogelijk gemaakt door"
+            },
+            "verification_code_sms": {
+                "description": "${generated_code} is uw eenmalige code om u aan te melden bij ${receiver_domain}\n\n @${receiver_domain} #${generated_code]"
```

Comment:
@onderay sorry should be 

```${generated_code} is uw eenmalige code om u aan te melden bij ${business_name}\n\n@${receiver_domain} #${generated_code}```

previous was missing `$business_name` and had a `]` at the end

Reply:
@DaveOrDead updated